lib/at_host/at_host.c

```diff
@@ -233,10 +233,12 @@ static int at_uart_init(char *uart_dev_name)
 		if (err) {
 			if (k_uptime_get_32() - start_time >
 			    CONFIG_AT_HOST_UART_INIT_TIMEOUT) {
+				LOG_ERR("UART check failed: %d. "
+				"AT host UART initialization timed out.", err);
 				return -EIO;
 			}
 
-			LOG_ERR("UART check failed: %d. "
+			LOG_INF("UART check failed: %d. "
```

Comment:
Should this be a `LOG_WRN` at least?

Reply:
I considered that, but since the behavior is as common as it is, I think that might be crying wolf. Since we don't coordinate the timing of the two chips on the Thingy:91 when initializing, and there is no HW flow control, the input to the UART is basically unknown for a while on startup. When the whole system is power cycled, this frequently happens.

Reply:
Thanks for explaining it.